testing/constants.ts

```diff
@@ -1,3 +1,12 @@
 export enum HttpStatusCode {
   Unauthorized = 401,
 }
+
+export const DEFAULT_ALLOWED_GET_DOMAINS = [
+  'coda-us-west-2-prod-blobs-upload.s3.us-west-2.amazonaws.com',
+  'coda-us-west-2-staging-blobs-upload.s3.us-west-2.amazonaws.com',
+  'coda-us-west-2-head-blobs-upload.s3.us-west-2.amazonaws.com',
+  'coda-us-west-2-adhoc-blobs-upload.s3.us-west-2.amazonaws.com',
+  'coda-us-west-2-dev-blobs-upload.s3.us-west-2.amazonaws.com',
```

Comment:
nit: might be nice to make this an array of regexes to handle all of these with one line

Reply:
Yeah, can we maybe do a regex check here? I think I'd rather not advertise all of these domains in this public repo.

Reply:
Since our docs only say codahosted, should we only allow codahosted for the cli?
![image](https://user-images.githubusercontent.com/100240943/175109660-2d03f102-6da7-4b73-a4b8-2f679051e66c.png)

Reply:
changed to regexes, let me know if we want to do this or just allow the "prod" env 

also tested that it works
![image](https://user-images.githubusercontent.com/100240943/175124214-b53648cc-6b59-4d09-b216-30820bde124c.png)